src/parser/Thread.co

```
@@ -20,6 +20,9 @@ module.exports = class Thread
     # least
     for post of @posts
       for quoted of post.quotelinks
-        @post[quoted]backlinks.push post.no
+        if @post[quoted]
+          that.backlinks.push post.no
+        else
+          console.log "can't find post[#quoted]"
```

Failing silently is fine. I broke backlinks in the last few commits unfortunately, sorry.